kani-driver/src/concrete_playback/test_generator.rs

```diff
@@ -231,8 +235,51 @@ impl KaniSession {
     }
 }
 
+fn gen_test_doc(harness: &HarnessMetadata, property: &Property) -> String {
+    let mut doc_str = match &harness.attributes.kind {
+        HarnessKind::Proof => {
+            format!("/// Test generated for harness `{}` \n", harness.pretty_name)
+        }
+        HarnessKind::ProofForContract { target_fn } => {
+            format!(
+                "/// Test generated for harness `{}` that checks contract for `{target_fn}`\n",
+                harness.pretty_name
+            )
+        }
+        HarnessKind::Test => {
+            unreachable!("Concrete playback for tests is not supported")
+        }
+    };
+    doc_str.push_str("///\n");
```

Comment:
Question: wouldn't be more efficient to use the `write!` macro? Do we even care? Can we generate multiple concrete tests or this will always just run once?

Reply:
I don't know if it's really faster, and I don't think we care. We run this once per test case that we generate.